config/webpack.prod.config.js

```diff
@@ -0,0 +1,42 @@
+const path = require('path');
+const merge = require('webpack-merge');
+const loaders = require('./webpack.common.config.js');
+const { CleanWebpackPlugin } = require('clean-webpack-plugin');
+
+const glob = require('glob');
+
+const rootPath = path.resolve(__dirname, '..');
+const srcDir = path.resolve(rootPath, 'src');
+
+const entryArray = glob.sync(path.join(srcDir, '**', '*.js'));
+const entryObject = entryArray.reduce((acc, item) => {
+    const name = path.relative(srcDir, item);
+    acc[name] = item;
+    return acc;
+}, {});
+
+/**
+ * Define all modules as external, so rollup won't bundle them together.
+ */
+const externals = {
+    yamlparser: true
+};
```

Comment:
question: why do you need this ?

Reply:
Seems that it was just a local issue with webpack on my side. Removed this code as a part of [commit](https://github.com/oat-sa/browser-support-checker/pull/1/commits/2d35b4d046c445891ca8e3d6ad4d0f5eb840e1a9)